apps/native-component-list/src/screens/Camera/Photo.web.tsx

```diff
@@ -1,4 +1,4 @@
-import { Ionicons } from '@expo/vector-icons';
+import Ionicons from '@expo/vector-icons/build/Ionicons';
```

Comment:
why `/build/`?

Reply:
cuz we don't export the types i.e. `FontAwesome.d.ts` https://github.com/expo/vector-icons